addons/nodot/interaction/Portal3D.gd

```diff
@@ -0,0 +1,67 @@
+#Write by: Z3R0_GT 
```

Comment:
Git will take care of this information so you can delete lines 1, 2 and 3. Don't worry, you will be credited.